website/content/docs/operations/index.mdx

```diff
@@ -0,0 +1,10 @@
+---
+layout: docs
+page_title: Operating Boundary
+description: |-
+  Operational tasks in Boundary.
+---
+
+# Maintaining and Operating Boundary
+
+Having visibility into various components of Boundary plays an important role in ensuring the health of its Controllers and Workers. While we intend to offer additional monitoring components in subsequent Boundary releases, Boundary can now export metrics for its Controllers and Workers which can be collected by Prometheus.  These metrics help you understand the current state of your Boundary deployment, thereby, enabling the diagnosis of deployed Controllers and Workers to help detect anomalies and mitigate. This section will cover the metrics available as well as the naming and labeling standards used for the metrics exported.
```

Comment:
```suggestion
Having visibility into various components of Boundary plays an important role in ensuring the health of its Controllers and Workers. While we intend to offer additional monitoring components in subsequent Boundary releases, Boundary can now export metrics for its Controllers and Workers which can be collected by Prometheus.  These metrics help you understand the current state of your Boundary deployment, thereby, enabling the diagnosis of deployed controllers and workers to help detect anomalies and mitigate. This section will cover the metrics available as well as the naming and labeling standards used for the metrics exported.
```

Reply:
I've seen these terms used as lowercase so changing to lowercase for consistency.

Reply:
Since we've added the health endpoint, I may modify the intro slightly to include a few sentences about that.